lib/config/index.js

```diff
@@ -74,6 +74,71 @@ exports.getLoaderConfig = function() {
   return cfg;
 };
 
+function updateNodelibTarget(target) {
```

Comment:
There should probably be a second argument here - `range` as boolean. We can then use `setVersion('^0.2.0)` for the package.json dependencies to ensure we get semver upgrade paths. While the depMap set will be range: false.